### PR TITLE
Prop css table

### PR DESCRIPTION
**Pull request by fedorenk0**

### CODE REVIEW

File: css/start-use-css/properties-in-css/table.html [hunk 1]
```diff
@@ -9,7 +9,51 @@
 </head>
 <body>
     <div>
-        
+        <h1 class="wrapper">Высота в таблице</h1>
+        <p class="first-p">Наследование позволяет задавать значения некоторых свойств единожды,
```
Comment (lutogin) [inline css/start-use-css/properties-in-css/table.html:13]: Use this format
```
<p class="first-p">
  Наследование позволяет задавать значения некоторых свойств единожды, определяя их для родителей 
  верхнего уровня.
</p>
```

File: css/start-use-css/properties-in-css/table.html [hunk 2]
```diff
@@ -9,7 +9,51 @@
 </head>
 <body>
     <div>
-        
+        <h1 class="wrapper">Высота в таблице</h1>
+        <p class="first-p">Наследование позволяет задавать значения некоторых свойств единожды, 
+            определяя их для родителей верхнего уровня.</p>
+        <h2 class="wrapper">Ширина в таблице</h2>
+        <p class="second-p">Самым низким приоритетом обладает стиль браузера — оформление,
```
Comment (lutogin) [inline css/start-use-css/properties-in-css/table.html:17]: same

File: css/start-use-css/properties-in-css/table.html [hunk 3]
```diff
@@ -9,7 +9,51 @@
...
+             которое по умолчанию применяется к элементам веб-страницы браузером.</p>
+    </div>
+    <div>         
+        <table border="5" widht="400" height="200">
```
Comment (lutogin) [inline css/start-use-css/properties-in-css/table.html]: Don't use properties for style description on an any tags. This property need use in CSS and plug their through class

**Review by lutogin — CHANGES_REQUESTED** [review]

File: css/start-use-css/properties-in-css/table.html (hunk 3 above)
Comment (fedorenk0) [reply css/start-use-css/properties-in-css/table.html]: corrected

File: css/start-use-css/properties-in-css/table.html (hunk 1 above)
Comment (fedorenk0) [reply css/start-use-css/properties-in-css/table.html:13]: corrected

File: css/start-use-css/properties-in-css/table.html (hunk 2 above)
Comment (fedorenk0) [reply css/start-use-css/properties-in-css/table.html:17]: corrected

**Review by lutogin — APPROVED** [review]

**Pull request merged**